code/game/objects/items/devices/forcefieldprojector.dm

```diff
@@ -16,6 +16,8 @@
 	var/max_fields = 3
 	var/list/current_fields
 	var/field_distance_limit = 7
+	var/creation_time = 10
```

Comment:
Document the new vars

Reply:
use time defines

Reply:
```suggestion
	/// Time it takes to materialize a forcefield.
	var/creation_time = 1 * SECONDS
```

I think this should probably be better?

Reply:
Documentation for both the new vars, and time defines used, appreciate the feedback.